test/e2e/configure_cni_test.go

```diff
@@ -64,6 +64,14 @@ func testConfigureCNI(t *testing.T) {
 	isConfiguredCorrectly, err := isCNIConfigured(t, kubeletLogPath)
 	require.NoError(t, err, "Error reading kubelet log")
 	assert.True(t, isConfiguredCorrectly, "CNI was not configured correctly")
+
+	// Kubelet arguments with paths that are set by bootstrapper
+	// Does not include node-labels and container-image since their paths do not depend on underlying OS
+	checkPathsFor := []string{"--bootstrap-kubeconfig", "--cloud-config", "--config", "--kubeconfig", "--log-file",
```

Comment:
Why are we testing this again?

Reply:
This time we are checking for all arguments that should be present after `configure-cni` is called.

Reply:
Sorry for not explaining it clearly, to be fair by the time we reach `configure-cni`, we assume bootstrap has been complete. Also, we can clearly see https://github.com/openshift/windows-machine-config-operator/pull/102/files#diff-1fb1151240e42debb7b16c5f09ca5afaR70 is superset of https://github.com/openshift/windows-machine-config-operator/pull/102/files#diff-564dc47aef3986957382c41ba37de57fR95, so I think you don't need to call it twice.

Reply:
How did you test this on the Windows node?

Reply:
I suggested this to @suhanime. I  don't see any harm in testing this again. It would guard against changes in `configure-cni` inadvertently changing the path separator in other kubelet args. But if you insist we can remove the non CNI args from this list.

Reply:
I suggest we keep it. The true spirit of e2e tests is to not write the tests according to the code, but to validate the end result. So the test shouldn't be written always assuming the other function would be run first. This will ensure at no point we are overwriting the other arguments when we add the cni arguments.
Secondly this can act as more robust way of testing if we need to drop certain arguments in the future.

Reply:
As discussed, changed `checkPaths` in `configure_cni_test.go` to only check for kubelet arguments added by `configure-cni`